Eatery/LookAheadViewController.swift

```diff
@@ -421,4 +424,17 @@ class LookAheadViewController: UIViewController, UITableViewDataSource, UITableV
         
         tableView.reloadData()
     }
+    
+    func currentMealIndex() -> Int {
+        let calendar = NSCalendar.currentCalendar()
+        let currentHour = calendar.component(.Hour, fromDate: NSDate())
+        if currentHour < LunchStartHour {
+            return 0 // Breakfast
+        } else if currentHour < DinnerStartHour {
+            return 1 // Lunch
+        } else {
+            return 2 // Dinner
+        }
+    }
```

Comment:
Alternatively:
```swift
func currentMealIndex() -> Int {
    let calendar = NSCalendar.currentCalendar()
    let currentHour = calendar.component(.Hour, fromDate: NSDate())
    switch currentHour {
    case 0...9: return 0 // Breakfast
    case 10...15: return 1 // Lunch
    default: return 2 // Dinner
    }
}
```

Also, I don't want to have private let properties if I only use them in this function.